addons/mail/models/mail_activity_type.py

```diff
@@ -125,3 +125,51 @@ def _inverse_triggered_next_type_id(self):
                 activity_type.chaining_type = 'trigger'
             else:
                 activity_type.chaining_type = 'suggest'
+
+    def write(self, values):
+        """ Protect some master types against deletiong or modification when they
```

Comment:
typo "deletiong" 

Actually not sure why we mention deletion inside the write.